### ISSUE TITLE
Recommend: Ai6win, Silky's engine .mes

### ISSUE THREAD

**Issue by Cosetto** [arcusmaximus/VNTranslationTools#12]
Can you implement these two engines? I got cpp code for it from the Internet, it dumps/inserts Silky's engine properly, creates line breaker with \n. The problem it can't dump Ai6win properly, some of the scripts are not extracted, can't insert too. I included the script of [link](https://vndb.org/v14826)(Ai6win) in the zip
[mestool.zip](https://github.com/arcusmaximus/VNTranslationTools/files/7784052/mestool.zip)

**Comment by arcusmaximus** [comment]
You mention "two engines", but going by the rest of the description I take it there's only one?

**Comment by Cosetto** [comment]
> You mention "two engines", but going by the rest of the description I take it there's only one?

They are different but the the script is same extension .mes

**Comment by arcusmaximus** [comment]
Initially I thought that "Ai6win" was some kind of shorthand for the AiShimai game, but after some investigation I found that it's indeed a different (older) version of the Silky's engine. While the ".mes" file extension is the same between the two, the format is ever so slightly different. I went ahead and implemented both of them.

Note that there's apparently also an "Ai5win" engine out there whose .mes format is completely different from Ai6win's and Silky's. That one is not implemented here.
